techscreenr-angular/client/app/questions/sidebar/sidebar.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Store } from '@ngrx/store';
+
+import { Question, Answer } from '../../../../models';
+
+@Component({
+  selector: 'app-question-sidebar',
+  templateUrl: './sidebar.component.html'
+})
+export class QuestionSidebarComponent {
+    @Input('questions') questions: Question[];
```

Comment:
@amartinelli12 You do not need to add 'questions' inside the @Input decorator function. You could use the code below and the component attribute would still be named 'questions'. Avoiding pointless aliasing is recommended in the [Angular.io ](https://angular.io/guide/styleguide#style-05-13)Style Guide.

@Input() questions